rust/Cargo.lock

```diff
@@ -1793,7 +1793,7 @@ dependencies = [
 [[package]]
 name = "payjoin_ffi"
 version = "0.18.0"
-source = "git+https://github.com/LtbLightning/payjoin-ffi?branch=v0.18.0#610aed9239c6e1e28cc56118e9b94b5000ae6052"
+source = "git+https://github.com/kumulynja/payjoin-ffi?branch=danger-local-https#8c02096df2ab333bf967f74cd09c001dd4b7852c"
```

Comment:
@kumulynja as `payjoin-ffi` has been updated with your latest changes. Pleas update the new commit hash. or branch.